manifests/g/Governikus/AusweisApp2/1.22.6/Governikus.AusweisApp2.installer.yaml

```diff
@@ -1,20 +1,20 @@
-# Created using wingetcreate 1.0.4.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
-
-PackageIdentifier: Governikus.AusweisApp2
-PackageVersion: 1.22.6
-Scope: machine
-InstallModes:
-- interactive
-- silent
-- silentWithProgress
-UpgradeBehavior: install
-Installers:
-- Architecture: x86
-  InstallerType: wix
-  InstallerUrl: https://www.ausweisapp.bund.de/fileadmin/user_upload/Software/AusweisApp2-1.22.6.msi
-  InstallerSha256: A5721665DC6F2E6E4DD944DBF631274047E599D1F85F47ECB4D2BF093C13D941
-  ProductCode: '{9093455B-A4E6-424A-8E71-CDD5000D9D89}'
-ManifestType: installer
-ManifestVersion: 1.1.0
-ReleaseDate: 2022-05-19
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Governikus.AusweisApp2
+PackageVersion: 1.22.6
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://www.ausweisapp.bund.de/fileadmin/user_upload/Software/AusweisApp2-1.22.6.msi
```

Comment:
```
  InstallerUrl: https://www.ausweisapp.bund.de/fileadmin/user_upload/Software/AusweisApp2-1.22.6.msi
```
404 Not Found

Reply:
![image](https://user-images.githubusercontent.com/11600822/187047046-c8b0ef72-4528-4ea3-8a6b-4708df7222d2.png)